validation/standard/init.go

```diff
@@ -7,16 +7,35 @@ import (
 	"github.com/BurntSushi/toml"
 )
 
-//go:embed standard-config-params-mainnet.toml standard-config-params-sepolia.toml standard-config-params-sepolia-dev-0.toml standard-config-roles.toml
+//go:embed *.toml
 var standardConfigFile embed.FS
 
 func init() {
 	Config = ConfigType{
-		Params: make(map[string]*Params),
-		Roles:  new(Roles),
+		Params:        make(map[string]*Params),
+		Roles:         new(Roles),
+		MultisigRoles: make(map[string]*MultisigRoles),
 	}
 
-	err := decodeTOMLFileIntoConfig("standard-config-roles.toml", Config.Roles)
+	err := decodeTOMLFileIntoConfig("standard-config-roles-universal.toml", Config.Roles)
+	if err != nil {
+		panic(err)
+	}
+
+	Config.MultisigRoles["mainnet"] = new(MultisigRoles)
+	err = decodeTOMLFileIntoConfig("standard-config-roles-mainnet.toml", Config.MultisigRoles["mainnet"])
+	if err != nil {
+		panic(err)
+	}
+
+	Config.MultisigRoles["sepolia"] = new(MultisigRoles)
+	err = decodeTOMLFileIntoConfig("standard-config-roles-sepolia.toml", Config.MultisigRoles["sepolia"])
+	if err != nil {
+		panic(err)
+	}
+
+	Config.MultisigRoles["sepolia-dev-0"] = new(MultisigRoles)
+	err = decodeTOMLFileIntoConfig("standard-config-roles-sepolia-dev-0.toml", Config.MultisigRoles["sepolia-dev-0"])
```

Comment:
To further cut down on redundant code related to `MultisigRoles` and `Params` you could loop through all network names like this:
```
	networks := []string{"mainnet", "sepolia", "sepolia-dev-0"}
	for _, network := range networks {
		Config.MultisigRoles[network] = new(MultisigRoles)
		err = decodeTOMLFileIntoConfig("standard-config-roles-"+network+".toml", Config.MultisigRoles[network])
		if err != nil {
			panic(err)
		}

		Config.Params[network] = new(Params)
		err = decodeTOMLFileIntoConfig("standard-config-params-"+network+".toml", Config.Params[network])
		if err != nil {
			panic(err)
		}
	}
```
And perhaps move the `panic` inside `decodeTOMLFileIntoConfig` to reduce lines even more

Reply:
[0a79b06](https://github.com/ethereum-optimism/superchain-registry/pull/300/commits/0a79b0608706551dee01073f16580ad75aaddf34)